app/backend/gwells/roles.py

```diff
@@ -57,6 +59,17 @@
     WELLS_EDIT_ROLE,
 )
 
+AQUIFERS_EDIT_ROLES = (
```

Comment:
You can drop AQUIFERS_EDIT_ROLES and AQUIFERS_VIEW_ROLES, this is actually leftover of how we used to do things.
We now check directly for a particular role, and rely on keycloak groups to apply the correct roles.